lib/index.js

```diff
@@ -56,12 +57,16 @@ module.exports = function(playlistURL, options) {
   }
 
   var tid;
+  var rpts;
+  var ytft = true;
```

Comment:
ytft means "YouTube First Time". This is required, as we have to let the m3u8 know if it's the first time. More information on this will be in my comment on the actual part that handles time.